BedrockConflictMetrics.h

```diff
@@ -0,0 +1,68 @@
+#include <libstuff/libstuff.h>
+#include <bitset>
+
+class BedrockConflictMetrics {
```

Comment:
This feels a little weird that there is a public class for a private data structure nobody else ever touches.  I could see a different way to encapsulate this like:

```
class BedrockConflictMetrics {
public:
    bool BedrockConflictMetrics::multiWriteOK(const string& commandName);
    ...

private:
    struct Command {
        void success();
        uint64_t _totalSuccessCount = 0;
        ...
    };

    static map<string, Command> _conflictInfoMap;
};
```

Not passionate about it.